Formula/quarantinecli.rb

```diff
@@ -0,0 +1,21 @@
+class Quarantinecli < Formula
+  desc "CLI Utility to quarantine and manage quarantined files on macOS"
+  homepage "https://github.com/Serena-io/quarantineCLI"
+  url "https://github.com/Serena-io/quarantineCLI/archive/refs/tags/1.0.0.tar.gz"
+  version "1.0.0"
+  sha256 "69ab41c71ed37c5b47e7dc5ed54bde28f15eaf38c81d804f768f74fb482a28e6"
+  license "MIT"
+
+  depends_on :macos
```

Comment:
Can we build this on Linux using the `swift` formula?

Reply:
Not at all, the tool uses LSQuarantine from CoreServices, a framework exclusive to Darwin platforms